test/scss/_config.scss

```diff
@@ -8,6 +8,18 @@ $fonts: (
     'italic': false,
   ),
 
+  'TestFont': (
+    'stack': ('Georgia', 'serif'),
+    'normal': (
+      'local': true,
+      'woff2': 'data:application/font-woff2;charset=utf-8;base64,…',
+      'woff': 'data:application/font-woff;charset=utf-8;base64,…',
+      'svgz': 'path-to-svg',
+    ),
+    'local': 'local-name' 'local name',
```

Comment:
@mirisuzanne So if I'm understanding this correctly... You can either set `local` as a top-level item on your font map, or you can nest it under an individual variant?

For locally-hosted fonts, this should be added both to the `specimen_variant` macro and the auto-added `@font-face` css. For web-hosted fonts, we only need to add it to the `specimen_variant` macro.

Currently, the only reason to list out variants in the font-map is to provide a path to local font files -- it's not used for web-hosted fonts. But now, it seems like we need to support it so people can provide `local` or `<format>` options? What if users don't want to list out each individual format?

Can you hop on a call to discuss?